File: APIs/py/universitat/models/alumne.py
from . import db

class Alumne(db.Model):
    DNI = db.Column(db.String(10), primary_key=True)
    nom = db.Column(db.String(50), nullable=False)
    email = db.Column(db.String(120), nullable=False)

    def __repr__(self):
        return f'<Alumne {self.nom}>'

File: APIs/.js/Node.js/universitat/models/Alumne.js
const mongoose = require('mongoose');

const alumneSchema = new mongoose.Schema({
  dni: { type: String, required: true, unique: true },
  nom: { type: String, required: true },
  edat: { type: Number, required: true },
});

module.exports = mongoose.model('Alumne', alumneSchema);
